data/sql_setup/prepare_schedule_a.sql

```diff
@@ -3,84 +3,298 @@ drop table if exists ofec_sched_a_nightly_retries;
 drop function if exists retry_processing_schedule_a_records(start_year integer);
 
 
--- Create queue tables to hold changes to Schedule A
-drop table if exists ofec_sched_a_queue_new;
-drop table if exists ofec_sched_a_queue_old;
-create table ofec_sched_a_queue_new as select * from fec_fitem_sched_a_vw limit 0;
-create table ofec_sched_a_queue_old as select * from fec_fitem_sched_a_vw limit 0;
-alter table ofec_sched_a_queue_new add column timestamp timestamp;
-alter table ofec_sched_a_queue_old add column timestamp timestamp;
-alter table ofec_sched_a_queue_new add column two_year_transaction_period smallint;
-alter table ofec_sched_a_queue_old add column two_year_transaction_period smallint;
-create index on ofec_sched_a_queue_new (sub_id);
-create index on ofec_sched_a_queue_old (sub_id);
-create index on ofec_sched_a_queue_new (timestamp);
-create index on ofec_sched_a_queue_old (timestamp);
-create index on ofec_sched_a_queue_new (two_year_transaction_period);
-create index on ofec_sched_a_queue_old (two_year_transaction_period);
-
--- Create trigger to maintain Schedule A queues for inserts and updates
+-- Create trigger to maintain Schedule A for inserts and updates
 -- These happen after a row is inserted/updated so that we can leverage pulling
 -- the new record information from the view itself, which contains the data in
 -- the structure that our tables expect it to be in.
-create or replace function ofec_sched_a_insert_update_queues() returns trigger as $$
+create or replace function ofec_sched_a_insert_update() returns trigger as $$
```

Comment:
Should we also keep in a `DROP FUNCTION` statement for the older function to make sure it gets removed?  Should that also be done at the time of the trigger recreation or just after?

*There's a note around those about this too.*

Reply:
Good catch! We should have code to drop the old functions.